03-transactions.md

```diff
@@ -1295,7 +1297,9 @@ These test the generation algorithm which all nodes use.
 These test the optional compact storage system. In many cases, an
 incorrect entry cannot be determined until its parent is revealed; an entry is
 specifically corrupted, along with all its children (except for the
-last test, which would require another eight samples to be detected). For
+last test, which would require another eight samples to be detected). 
+[ FIXME: I can't tell what's intended here. When you say "an entry is specifically corrupted ..." do you mean that an entry that can't be detected is specifically corrupted ... or ...? And how does that parenthetical "except" relate? ]
+For
```

Comment:
Yes, I'm not sure what this is saying either.  Remove the parenthetical comment and there's no loss.